### DISCUSSION TITLE
Getting Mesh From Jukkapalm

### DISCUSSION THREAD

**Discussion by Tduck5819** [piotrulos/MSCModLoader#190] [Help me] [answered]
I am trying to get the mesh from the gameobject jukkapalm i tried to use asset ripper with no succes i only got the scene file by exporting the jukkapalm from level2. Does someone already have the mesh or can someone tell me how i can get it?

**Comment by piotrulos** [comment, marked as answer]
Use asset studio, asset ripper doesn't correctly export stuff
